static/index.html

```diff
@@ -7,25 +7,29 @@
     <link href="out/styles/app.css" rel="stylesheet" />
     <link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png" />
     <link rel="icon" type="image/png" sizes="32x32" href="media/favicon.png" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" />
   </head>
-
   <body>
     <background>
-      <div id="pre-grid"></div>
-      <div id="grid"></div>
-      <header
-        ><a href="http://ubq.fi/">
+      <div id="pre-grid"> </div>
```

Comment:
the formatting looks messed

Reply:
> the formatting looks messed

That's the prettier config, was way messier before now because the config hasn't been in use